frontend/src/redux/selectors/resource.js

```diff
@@ -1,49 +1,45 @@
 import { resourceEnum, sortFieldEnum } from "../../utils/enums";
-
+import { createSelector } from "reselect";
 const resourceName = (resource) =>
   (resource.type === resourceEnum.INDIVIDUAL
     ? resource.contactName
-    : resource.companyName) || "";
+    : resource.companyName) ?? "";
 
-const resourceLocation = (resource) => resource.address || "";
+const resourceLocation = (resource) => resource.address ?? "";
```

Comment:
Nullish coalescing because I'm not sure if we can trust there won't be a weird value like `"0"` for address ;)